sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v2/schema.yaml

```diff
@@ -0,0 +1,57 @@
+fields:
+- mode: NULLABLE
+  name: date
+  type: DATE
+  description: Date of the visit
+- mode: NULLABLE
+  name: visit_identifier
+  type: STRING
+  description: Visit Identifier - Uniquely identifies a visit; concatenation of user_pseudo_id and ga_session_id
+- mode: NULLABLE
+  name: device_category
+  type: STRING
+  description: Device Category - The device category the visitor used to visit the site
+- mode: NULLABLE
+  name: operating_system
+  type: STRING
+  description: Operating System - The operating system the visitor used to visit the site
+- mode: NULLABLE
+  name: language
+  type: STRING
+  description: Language - Language of visitor's device
+- mode: NULLABLE
+  name: country
+  type: STRING
+  description: Country - The country from which events were reported, based on IP address
+- mode: NULLABLE
+  name: source
+  type: STRING
+  description: Source - Referring partner domain
+- mode: NULLABLE
+  name: medium
+  type: STRING
+  description: Medium - Category of the source, such as 'organic' for a search engine
+- mode: NULLABLE
+  name: campaign
+  type: STRING
+  description: Campaign - Identifier for the marketing campaign
+- mode: NULLABLE
+  name: ad_content
+  type: STRING
+  description: Ad Content - Indicates the particular link within a campaign
+- mode: NULLABLE
+  name: browser
+  type: STRING
+  description: Browser
+- mode: NULLABLE
+  name: download_events
+  type: INT64
+  description: Download Events
+- mode: NULLABLE
+  name: downloads
+  type: INT64
+  description: Downloads
+- mode: NULLABLE
+  name: non_fx_downloads
+  type: INT64
+  description: Non Firefox Downloads
```

Comment:
The descriptions for these metrics would be a great place to document the context/caveats you mentioned to me in Slack.

Also, the more readable `INTEGER` can be used here instead of `INT64`.

Reply:
Good to know, thanks!